### ISSUE TITLE
styling is not getting applied when siemensIX component is used on lit 

### ISSUE THREAD

**Issue by kcmyprojects** [siemens/ix#632]
### What happened?

we are using siemensIX code and coverting it into lit.
general observation is when component is created in separate file and then its template is used in another file, then styling is not getting applied.
for eg: this can be seen on siemensIX component **data time picker** and **HTML table**.

an example code is written in stackBliz  https://stackblitz.com/edit/hzcmgg?file=src%2Findex.html.  which shows styling is not getting applied on **date time picker**

Request you to please let us know how to handle above issue.

### What type of frontend frameware are you seeing the problem on?

Others

### Which version of iX do you use?

1.1.0

### Code to produce this issue.

```shell
https://stackblitz.com/edit/hzcmgg?file=src%2Findex.html. 
in the above link, test.js file is newly added and in the index.html <my-element></my-element> is added from the file test.js
```

**Comment by danielleroux** [comment]
@kcmyprojects Lit is using shadow-dom (https://lit.dev/docs/components/shadow-dom/). If you want to use the components with iX v1 you have to deactive the shadow-dom or load the complete css every time.

With v2 we changing this behavior then we are fully encapsulated and you can use every component inside shadow-dom

**Comment by danielleroux** [comment]
Will be solved with #511

**Comment by h4de5** [comment]
@kcmyprojects have a look here: https://stackblitz.com/edit/hzcmgg-mduce2?file=src%2Ftest.js,src%2Findex.html
not the cleanest solution, but adding 3 lines until the new version is out could be ok.
